kratos/solving_strategies/strategies/line_search_strategy.h

```diff
@@ -147,7 +151,44 @@ class LineSearchStrategy
         bool ReformDofSetAtEachStep = false,
         bool MoveMeshFlag = false
     ): ResidualBasedNewtonRaphsonStrategy<TSparseSpace, TDenseSpace, TLinearSolver>(model_part, pScheme, pNewLinearSolver,pNewConvergenceCriteria,pNewBuilderAndSolver,MaxIterations,CalculateReactions,ReformDofSetAtEachStep, MoveMeshFlag)
-    {}
+    {
+        Parameters default_settings = GetDefaultSettings();
+        this->mSettings.AddMissingParameters(default_settings);
+        AssignSettings();
+    }
+
+    LineSearchStrategy(
+        ModelPart& model_part,
+        typename TSchemeType::Pointer pScheme,
+        typename TLinearSolver::Pointer pNewLinearSolver,
+        typename TConvergenceCriteriaType::Pointer pNewConvergenceCriteria,
+        Parameters Settings,
+        bool CalculateReactions = false,
```

Comment:
the last two bools should be included in the parameters

Reply:
done